server/libi2dx.py

```diff
@@ -27,27 +27,30 @@ def get(self):
 
 class I2DXWebSocket(websocket.WebSocketHandler):
 
-	def allow_draft76(true):
+	def allow_draft76(self):
 		return True
 
-	def open(self):
+	def open(self, *args, **kwargs):
 		self.write_message("Ready")
-		self.stream.socket.setsockopt(socket.SOL_TCP, socket.TCP_NODELAY, 0)
-		print "i2dx connection opened!"
+		self.ws_connection.stream.socket.setsockopt(socket.SOL_TCP, socket.TCP_NODELAY, 0) # type: ignore
```

Comment:
[NOTE]
`self.ws_connection.stream.socket.setsockopt` causes an error in lint, but there is no problem with this process, so I ignored it.